static/js/components/EducationForm.js

```diff
@@ -138,7 +189,9 @@ class EducationForm extends ProfileFormFields {
           <Cell col={7} />
           <Cell col={1}>
             <Switch
-              onChange={()=>{this.handleSwitchClick(level.value);}}
+              id={`profile-tab-education-switch-${level.value}`}
+              onClick={()=>{this.handleSwitchClick(level.value);}}
+              readOnly={true}
```

Comment:
I experimented with this and `TestUtils.Simulate.change` should work fine without a second argument. This does work for me as is but if this fixes the testing issue I'd prefer to use `onChange` since it's documented in the `react-mdl` docs